src/bin/generate_test_data.rs

```diff
@@ -3,11 +3,11 @@ use nmg_league_bot::db::{raw_diesel_cxn_from_env, run_migrations};
 use nmg_league_bot::models::brackets::{BracketType, NewBracket};
 use nmg_league_bot::models::player::NewPlayer;
 use nmg_league_bot::models::player_bracket_entries::NewPlayerBracketEntry;
-use nmg_league_bot::models::season::{NewSeason, Season};
+use nmg_league_bot::models::season::{NewSeason, Season, SeasonState};
 
 extern crate dotenv;
 
-const PLAYERS_PER_BRACKET: i32 = 16;
+const PLAYERS_PER_BRACKET: i32 = 8;
```

Comment:
this is kind of arbitrary b/c of what i was working on, but idc. we can make it configurable if it ever really matters